src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java

```diff
@@ -125,44 +147,62 @@ protected JacobianMatrix createJacobianMatrix(EquationSystem equationSystem, Vol
 
         private Status status = Status.VALID;
 
-        public LfSensitivityFactor(SensitivityFactor factor, LfNetwork lfNetwork) {
+        public LfSensitivityFactor(SensitivityFactor factor, Network network, LfNetwork lfNetwork, EquationSystem equationSystem) {
             this.factor = factor;
             if (factor instanceof BranchFlowPerInjectionIncrease) {
                 BranchFlow branchFlow = ((BranchFlowPerInjectionIncrease) factor).getFunction();
                 functionLfBranch = lfNetwork.getBranchById(branchFlow.getBranchId());
                 equationTerm = functionLfBranch != null ? (EquationTerm) functionLfBranch.getP1() : null;
+                functionLfBus = null;
             } else if (factor instanceof BranchFlowPerPSTAngle) {
                 BranchFlow branchFlow = ((BranchFlowPerPSTAngle) factor).getFunction();
                 functionLfBranch = lfNetwork.getBranchById(branchFlow.getBranchId());
                 equationTerm = functionLfBranch != null ? (EquationTerm) functionLfBranch.getP1() : null;
+                functionLfBus = null;
             } else if (factor instanceof BranchIntensityPerPSTAngle) {
                 BranchIntensity branchIntensity = ((BranchIntensityPerPSTAngle) factor).getFunction();
                 functionLfBranch = lfNetwork.getBranchById(branchIntensity.getBranchId());
                 equationTerm = functionLfBranch != null ? (EquationTerm) functionLfBranch.getI1() : null;
+                functionLfBus = null;
             } else if (factor instanceof BranchFlowPerLinearGlsk) {
                 BranchFlow branchFlow = ((BranchFlowPerLinearGlsk) factor).getFunction();
                 functionLfBranch = lfNetwork.getBranchById(branchFlow.getBranchId());
                 equationTerm = functionLfBranch != null ? (EquationTerm) functionLfBranch.getP1() : null;
+                functionLfBus = null;
+            } else if (factor instanceof BusVoltagePerTargetV) {
+                functionLfBranch = null;
+                Bus bus = ((BusVoltagePerTargetV) factor).getFunction().getBusRef().resolve(network).orElseThrow();
+                functionLfBus = lfNetwork.getBusById(bus.getId());
+                equationTerm = equationSystem.getEquationTerm(ElementType.BUS, functionLfBus.getNum(), EquationTerm.VariableEquationTerm.class);
```

Comment:
@geofjamg @annetill I would like to do the same thing as the branches (like using functionLfBus.getV() to get an Evaluable), but I am not sure if the Evaluable should be the bus_V equation, or the bus_V equation term ? Moreover, currently, getV() returns v / nominalV, so I am not sure whether the nominal V should be included in the Evaluable or not

Reply:
You can use the BUS_V equation term.
No the / nominalV should not be included and just be careful to add it when updating the IIDM network (LfBusImpl.updateState method)